tasks/eth/ink-003-fp-holocene-pectra-upgrade/README.md

```diff
@@ -0,0 +1,67 @@
+# Holocene Hardfork Upgrade + L1 Pectra Support Upgrade
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts for the Holocene hardfork as per Upgrade Proposal #11.
+
+This also utilizes the absolute prestate `0x035ac388b5cb22acf52a2063cfde108d09b1888655d21f02f595f9c3ea6cbdcd` that enables L1 pectra support, to be included in Upgrade Proposal #12.
+
+The proposal #11 was:
+
+- [X] [Posted](https://gov.optimism.io/t/upgrade-proposal-11-holocene-network-upgrade/9313) on the governance forum.
+- [X] [Approved](https://vote.optimism.io/proposals/20127877429053636874064552098716749508236019236440427814457915785398876262515) by Token House voting.
+- [X] Not vetoed by the Citizens' house.
+- [X] Executed on OP Mainnet.
+
+The proposal #12 was:
+
+- [X] [Posted](https://gov.optimism.io/t/upgrade-proposal-12-l1-pectra-readiness/9706) on the governance forum.
+- [x] Posted on the [governance forum](https://gov.optimism.io/t/upgrade-proposal-12-l1-pectra-readiness/9706).
+- [x] Approved by [Token House voting](https://vote.optimism.io/proposals/38506287861710446593663598830868940900144818754960277981092485594195671514829).
+- [ ] Not vetoed by the Citizens' house.
+- [ ] Executed on OP Mainnet.
+
+The governance proposal should be treated as the source of truth and used to verify the correctness of the onchain operations.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.8.0-rc.4](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.4) release.
+
+## Pre-deployments
+
+- `MIPS` - `0x5fE03a12C1236F9C22Cb6479778DDAa4bce6299C`
```

Comment:
Previous one is `0xaa59a0777648bc75cd10364083e878c1ccd6112a`

Reply:
I see, so we'll want to utilize the shared MIPs contract

Reply:
This is incorrect we stay with this version.

Reply:
so this task is all good?